stable_baselines3/common/policies.py

```diff
@@ -885,6 +895,7 @@ def __init__(
         normalize_images: bool = True,
         optimizer_class: Type[th.optim.Optimizer] = th.optim.Adam,
         optimizer_kwargs: Optional[Dict[str, Any]] = None,
+        _init_optimizer=True,  # Currently unused, see PR #1771
```

Comment:
@araffin I'm currently testing enabling this one too. I will update this PR accordingly, hence switching it back to draft.